src/ol/renderer/canvas/TileLayer.js

```diff
@@ -74,62 +168,109 @@ class CanvasTileLayerRenderer extends CanvasLayerRenderer {
      */
     this.renderedTiles = [];
 
+    /**
+     * @private
+     * @type {string}
+     */
+    this.renderedSourceKey_;
+
+    /**
+     * @private
+     * @type {number}
+     */
+    this.renderedSourceRevision_;
+
     /**
      * @private
      * @type {boolean}
      */
-    this.newTiles_ = false;
+    //this.newTiles_ = false;
```

Comment:
Should this be removed or uncommented?

Reply:
Since the rest of the code `this.newTiles_` is commented out, I left it like this. But all that could be removed - it's no longer needed.